knotx-core/src/main/java/com/cognifide/knotx/repository/RepositoryType.java

```diff
@@ -44,7 +46,7 @@ public boolean validate(RepositoryConfiguration.RepositoryMetadata metadata) {
         public Repository<String, URI> create(RepositoryConfiguration.RepositoryMetadata metadata,
                                               KnotxVerticle verticle) {
             return RemoteRepository.of(metadata.getPath(), metadata.getDomain(), metadata.getPort(), verticle
-                    .getVertx().createHttpClient());
+                    .getVertx().createHttpClient(new HttpClientOptions().setTryUseCompression(true)));
```

Comment:
I dont't think we need to have it configurable. As the method says it tries to use compression. if it's not supported it still work as expected.